### PR TITLE
New Chess Cheat Sheet

### PR DESCRIPTION
**Pull request by samskeller**
**What does your Instant Answer do?**
This Instant Answer provides basic information on the different pieces in chess and how they move, basic gameplay, how the game ends, and special movements like castling.

**What is the data source for your Instant Answer? (Provide a link if possible)**
Wikipedia: https://en.wikipedia.org/wiki/Chess

**Why did you choose this data source?**
Because it had the most comprehensive and yet still readable guide on how the game is played as we as additional resources if the user wants to read more.

**Are there any other alternative (better) data sources?**
Chess.com has a pretty good explanation as well (http://www.chess.com/learn-how-to-play-chess) but I thought the Wikipedia page would give better context.

**Which communities will this Instant Answer be especially useful for? (gamers, book lovers, etc)**
People who love board games and want to learn chess!

**Is this Instant Answer connected to a DuckDuckHack [Instant Answer idea](https://duck.co/ideas)?**
Yes -- https://duck.co/ideas/idea/4677/chess-pieces

**Are you having any problems? Do you need our help with anything?**
Nope!

**Where did you hear about DuckDuckHack? (For first time contributors)**
Stumbled across it when looking for how to help out my favorite search engine!

**What does the Instant Answer look like? (Provide a screenshot for new or updated Instant Answers)**
screen shot 2015-09-10 at 10 26 56 pm

-----
IA Page: https://duck.co/ia/view/chess_cheat_sheet

### CODE REVIEW

File: share/goodie/cheat_sheets/json/chess.json [hunk 1]
```diff
@@ -0,0 +1,75 @@
+{
+    "id": "chess_cheat_sheet",
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/chess.json]: Let's change this to `chess_pieces_cheat_sheet`

Comment (moollaza) [inline share/goodie/cheat_sheets/json/chess.json]: Whoops, just realized this covers a bit more than pieces. Alright I'll update the IA page and ID to be `chess_cheat_sheet`

File: share/goodie/cheat_sheets/json/chess.json [hunk 2]
```diff
@@ -0,0 +1,75 @@
+{
+    "id": "chess_cheat_sheet", 
+    "name": "Chess",
+    "description": "Strategy Board Game", 
+    "metadata": {
```
Comment (moollaza) [inline share/goodie/cheat_sheets/json/chess.json]: Let's add some aliases so we can make the triggering a little more flexible:

```json
"aliases": [
    "chess pieces",
    "chess gameplay",
    "chess movement"
]
```

Then queries like "chess gameplay cheatsheet" or "chess movement reference" will trigger this.

**Comment by moollaza** [comment]
@samskeller thanks a lot for contributing! (we really appreciate it :smiley:) I've left you some feedback that needs to be implemented. Once things are updated please let us know by leaving a comment, so we can review this again. If you have any questions, feel free to ask them here!

**Comment by samskeller** [comment]
@moollaza thanks for the feedback! I've committed a change to add aliases. Let me know if there's anything else to do!

**Comment by moollaza** [comment]
@samskeller this LGTM :+1:

Thanks a lot for contributing (we really appreciate it :smiley:). We invite you to stick around and comment/help on other PR's & Issues or even submit an Instant Answer!

**Comment by samskeller** [comment]
Thanks @moollaza -- I actually already started and tackled <a href="https://github.com/duckduckgo/zeroclickinfo-spice/issues/2158#event-409792332">an issue</a> last night!

**Comment by moollaza** [comment]
@samskeller congrats! It's live: https://duckduckgo.com/?q=chess+cheat+sheet&ia=cheatsheet

Thanks again for taking the time to contribute! We really appreciate it. Feel free to stick around and comment/help on other PR's, tackle open issues or even submit more Instant Answers!

**Comment by moollaza** [comment]
By the way, we think you did a great job with this! Are you interested in helping others who build similar Instant Answers in the future?

If yes, we'll reach out and add you to a private list of mentors as well as send over some DDG merch as an additional thank you. Just email <open@duckduckgo.com> with the subject 'I want to help.' Please include your shipping address, phone-number (required for shipping outside USA) and t-shirt size. We'll reach out from there!

Otherwise, thank you for this amazing contribution. We hope you'll keep exploring and playing around with DuckDuckHack.

**Comment by samskeller** [comment]
@moollaza thanks! I'm interested in what it means to be a private mentor -- what responsibilities does that come with?

**Comment by moollaza** [comment]
@samskeller we're in the process of figuring out what it would involve, we're thinking we'd try to connect whichever mentors are available at the time with a developer who needs assistance. But we'd like to work with everyone to make sure it's useful and we're open to idea. You're not committing to anything, and there are no requirements. We're preparing some documentation which will make available once it's ready

**Pull request merged**